cmd/cli/kubectl-kyverno/commands/test/test.go

```diff
@@ -98,6 +98,52 @@ func runTest(out io.Writer, testCase test.TestCase, registryAccess bool, auditWa
 	} else {
 		fmt.Fprintln(out, "  Applying", len(policies)+len(vaps), pluralize.Pluralize(len(policies)+len(vaps), "policy", "policies"), "to", len(uniques), pluralize.Pluralize(len(uniques), "resource", "resources"), "...")
 	}
+	policyToUnknownRules := make(map[string]string)
+	var policyerr string
+	for _, res := range testCase.Test.Results {
+		if res.IsValidatingAdmissionPolicy {
+			continue
+		}
```

Comment:
In case the policy is of type `ValidatingAdmissionPolicy`, we need to check that the policy name specified in the results matches one of the policies provided from the files. They have no rules. We only have to check the policy name. Is it possible to do that?